### PR TITLE
Added the create and update notice for the fuctions

### PR DESCRIPTION
**Pull request by gautam071**
Targets #1369, 1351

### Description

- Added the created Invoice, Prospects notice. 
- Added the Updated Invoice, and Prospects notice as well.
### Checklist:

- [x] I have performed a self-review of my code.

### CODE REVIEW

File: Modules/Prospect/Http/Controllers/ProspectController.php [hunk 1]
```diff
@@ -74,7 +74,7 @@ public function update(Request $request, $id)
     {
         $data = $this->service->update($request->all(), $id);
 
-        return redirect($data['route']);
+        return redirect($data['route'])->with('status', 'Prospect created/updated  succesfully!');
```
Comment (GauravGusain98) [inline Modules/Prospect/Http/Controllers/ProspectController.php:77]:
```suggestion
        return redirect($data['route'])->with('status', 'Prospect created/updated successfully!');
```

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

File: Modules/Prospect/Http/Controllers/ProspectController.php (hunk 1 above)
Comment (GauravGusain98) [reply Modules/Prospect/Http/Controllers/ProspectController.php:77]: @gautam071 Just a minor fix, else looks good

Comment (gautam071) [reply Modules/Prospect/Http/Controllers/ProspectController.php]: @GauravGusain98 Done

**Pull request merged**